tests/integration/create_cluster/gce_byo_sa/options.yaml

```diff
@@ -0,0 +1,9 @@
+ClusterName: ha-gce.example.com
+Zones:
+- us-test1-a
+MasterZones:
+- us-test1-a
+Cloud: gce
+KubernetesVersion: v1.15.6-beta.1
+Project: testproject
+GCEServiceAccount: test-account@testproject.iam.gserviceaccount.com
```

Comment:
I think you need to add this `gce_byo_sa` test case to `cmd/kops/create_cluster_integration_test.go`. I'd also expect to see `test-account@testproject.iam.gserviceaccount.com` in some expected output values for this test somewhere... in a kubernetes.tf or in `expected-v1alpha2.yaml` ?